index.html

```diff
@@ -5,16 +5,29 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Portfolio Website</title>
+    <title>Portfolio Project Website</title>
```

Comment:
* Poor titles:
  * *Microverse project | Your portfolio.*
  * *Portifolio project*
* Better page titles:
  * *Mark's portifolio*
  * *Mark full-stack webdeveloper*
 Kindly make the necessary adjustments to the title.